models/iaf_psc_alpha.nestml

```diff
@@ -84,32 +84,32 @@ neuron iaf_psc_alpha:
   end
 
   equations:
-    shape I_shape_in = pA * (e/tau_syn_in) * t * exp(-1/tau_syn_in*t)
-    shape I_shape_ex = pA * (e/tau_syn_ex) * t * exp(-1/tau_syn_ex*t)
+    shape I_shape_in = (e / tau_syn_in) * t * exp(-t / tau_syn_in)
```

Comment:
so do we now enforce the fact that ``shape`` objects have no dimension? if so it should appear in the documentation (and be properly checked)